Filter/DataExtractor/Method/ValueKeysExtractionMethod.php

```diff
@@ -53,9 +53,6 @@ public function extract(FormInterface $form)
                         $values['value'][$key][$k] = $v;
                     }
                 }
-            } else {
-                dump($keys, $data);
-                throw new \InvalidArgumentException(sprintf('No value found for key "%s" in form data.', $key));
```

Comment:
You have removed the exception, I'm not sure about this

Reply:
Yes, because calling 

`$this->filterUpdater->addFilterConditions($filter, $filterBuilder);`

will throw if we do not provide all the date filters. I think filters are optional and  should not throw an exception if one is missing. All the other extraction method have the same behavior and it is not part of [the contract](https://github.com/SpiriitLabs/form-filter-bundle/blob/880f58a390c11a637e0298966a029b1b647673fd/Filter/DataExtractor/FormDataExtractorInterface.php#L30-L37) either.